### ISSUE TITLE
Can't discard duplicate rules

### ISSUE THREAD

**Issue by Lin07ux** [ben-eb/cssnano#86]
**code:**
body {
  padding: 50px;
}

body {
	padding: 0;
}

**results:**
body{padding:50px;padding:0}

What can I do to get the results as:
body {padding:0}

**Comment by ben-eb** [comment]
Currently it's not supported; I need more data before I can do this feature well. Not sure when I'll have time for this as my open source time is very limited these days.

**Comment by jakefreeberg** [comment]
:+1: on this issue... gulp-minify-css didn't have this problem. Having duplicate styles like this is basically broken/poorly written css, but if you could protect us from ourselves or the junior members of our teams, that would be great.

**Comment by ben-eb** [comment]
Yep, have had a few enquires about this, and a pull request which didn't quite have the robustness that I am looking for this feature. It's coming, but any help would be greatly appreciated.

https://github.com/ben-eb/postcss-discard-duplicates/pull/4

**Comment by ben-eb** [comment]
If you're interested in helping this issue move forward, you can help out by contributing to https://github.com/ben-eb/css-values, which we will be using to validate properties before merging.
